docs/UserGuide.md

```diff
@@ -614,7 +614,14 @@ You can paste the command into the command box after step 2 and repeat step 1-2
 <br>
 
 **Q**: I double-clicked the productivenus.jar file but the app is not starting. What should I do?<br>
-**A**: <br>
+**A**: This is most likely caused by the `security and privacy setting` of your operating system. You can either
+modify your `security and privacy setting` (depends on your operating system) or run ProductiveNUS with the `command
+prompt`.
+
+Below are the instructions for using the `command prompt` to open ProductiveNUS for Windows Users.
+Ubuntu and MacOS follow the similar instructions but with a different way of opening the `command prompt` and
```

Comment:
How about 'Mac uses the Terminal application to access its command prompt while ubutu.....'. It can be made more user centric by adding 'If your computer runs with  Ubuntu or MacOS, the following instructions is applicable but do note that....'